lib/APIPlugin.js

```diff
@@ -56,7 +55,6 @@ class APIPlugin {
 				/**
 				 *
 				 * @param {Parser} parser webpack parser
-				 * @returns {void}
```

Comment:
https://github.com/webpack/webpack/issues/7122 - section Return!?

Reply:
hmm yes the guide was wrong `@returns {void}` can't be omitted. Omitting it would mean `@returns {any}`.

Reply:
ok, will have to add them back then... in all PR's. This one is done.